glustercli/cmd/volume.go

```diff
@@ -233,36 +233,52 @@ var volumeDeleteCmd = &cobra.Command{
 }
 
 var volumeGetCmd = &cobra.Command{
-	Use:   "get",
+	Use:   "get <VOLNAME|all> <key|all>",
 	Short: helpVolumeGetCmd,
 	Args:  cobra.ExactArgs(2),
 	Run: func(cmd *cobra.Command, args []string) {
 
+		table := tablewriter.NewWriter(os.Stdout)
+		table.SetHeader([]string{"Volume Name", "Name", "Modified", "Value", "Default Value", "Option Level"})
+		table.SetAlignment(tablewriter.ALIGN_LEFT)
+
 		volname := args[0]
 		optname := args[1]
 
-		opts, err := client.VolumeGet(volname, optname)
-		if err != nil {
-			log.WithError(err).Error("error getting volume options")
-			failure("Error getting volume options", err, 1)
+		var volList []string
+		if volname == "all" {
```

Comment:
what happens if the user creates a volume with the name `all` and want to fetch the volume option for that particular volume :D.

Reply:
should I add this to volume-create?